moveit2_galactic.repos

```diff
@@ -1,5 +1,9 @@
 # TODO(henningkayser): Remove once ros2_control is released (https://github.com/ros-controls/ros2_control/issues/421)
 repositories:
+  moveit_resources:
+    type: git
+    url: https://github.com/JafarAbdi/moveit_resources
+    version: pr-moveit_configs_utils
```

Comment:
Because of how CI works now if you put something in moveit2.repos you don't also need it here.  Robert (using some shell scripting magic) made so the ros version-specific repos files extend the base one.  Putting it here might be the cause of some of the CI failures.